src/views/Stake/components/StakeArea/components/StakeInputArea/StakeInputArea.tsx

```diff
@@ -67,6 +67,11 @@ export const StakeInputArea: React.FC<{ isZoomed: boolean }> = props => {
 
   return (
     <Box mb={3}>
+      {currentAction === "UNSTAKE" && (
+        <InfoNotification>
+          <Trans>Inverse bonds are live & provide a greater discount than spot prices on the open market</Trans>
+        </InfoNotification>
+      )}
```

Comment:
I think this would be better if it checked if Inverse Bonds were actually live. That way we never have to turn it off.